cni/network/invoker_cns.go

```diff
@@ -127,97 +132,167 @@ func (invoker *CNSIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, erro
 			hostSubnet:         response.PodIPInfo[i].HostPrimaryIPInfo.Subnet,
 			hostPrimaryIP:      response.PodIPInfo[i].HostPrimaryIPInfo.PrimaryIP,
 			hostGateway:        response.PodIPInfo[i].HostPrimaryIPInfo.Gateway,
-		}
-
-		// set the NC Primary IP in options
-		// SNATIPKey is not set for ipv6
-		if net.ParseIP(info.ncPrimaryIP).To4() != nil {
-			addConfig.options[network.SNATIPKey] = info.ncPrimaryIP
+			addressType:        response.PodIPInfo[i].AddressType,
+			macAddress:         response.PodIPInfo[i].MacAddress,
+			isDefaultInterface: response.PodIPInfo[i].IsDefaultInterface,
+			routes:             response.PodIPInfo[i].Routes,
 		}
 
 		log.Logger.Info("Received info for pod",
-			zap.Any("ipv4info", info),
-			zap.Any("podInfo", podInfo))
-		ip, ncIPNet, err := net.ParseCIDR(info.podIPAddress + "/" + fmt.Sprint(info.ncSubnetPrefix))
-		if ip == nil {
-			return IPAMAddResult{}, errors.Wrap(err, "Unable to parse IP from response: "+info.podIPAddress+" with err %w")
-		}
+			zap.Any("ipinfo", info),
+			zap.Any("podInfo", podInfo),
+			zap.String("component", "cni-invoker-cns"))
```

Comment:
no need to add component in every log.. check with paulyu on this

Reply:
removed after rebase